### ISSUE TITLE
LFG window bug

### ISSUE THREAD

**Issue by Xrhoys** [Foglio1024/Tera-custom-cooldowns#143]
After replacing S1UI partyboard with empty version, LFG window doesn't show up at all  while it works normally with the original UI enabled.

**Comment by Foglio1024** [comment]
Use Ctrl-Y instead of Y only.

**Comment by Xrhoys** [comment]
change shortcut to ctrl Y?

**Comment by Foglio1024** [comment]
No, TCC already has Ctrl-Y shortcut.

**Comment by Xrhoys** [comment]
doesnt work

**Comment by Foglio1024** [comment]
Make sure that you have the latest tcc-stub and restart proxy.

**Comment by Xrhoys** [comment]
yup i just noticed you updated the proxy module. Thanks and sorry for the trouble.
